Increase the initial mark stack size to avoid stack overflows, since
these inhibit parallel marking (see GC_mark_some()).

diff --git a/mark.c b/mark.c
index 50e6cfd7..907f7763 100644
--- a/mark.c
+++ b/mark.c
@@ -66,7 +66,7 @@ GC_INNER struct obj_kind GC_obj_kinds[MAXOBJKINDS] = {
 };
 
 # ifndef INITIAL_MARK_STACK_SIZE
-#   define INITIAL_MARK_STACK_SIZE (1*HBLKSIZE)
+#   define INITIAL_MARK_STACK_SIZE (256*HBLKSIZE)
                 /* INITIAL_MARK_STACK_SIZE * sizeof(mse) should be a    */
                 /* multiple of HBLKSIZE.                                */
                 /* The incremental collector actually likes a larger    */
